lib/migration.pm

```diff
@@ -114,7 +114,7 @@ sub deregister_dropped_modules {
     for my $name (grep($_, split(/,/, $droplist))) {
         record_info "deregister $name", "deregister $name module and remove it from SCC_ADDONS";
         if ($name eq 'ltss') {
-            if (check_var('SLE_PRODUCT', 'hpc')) {
+            if ((check_var('SLE_PRODUCT', 'sles') && check_var('CROSS_PRODUCT_MIGRATION', 'from_hpc')) || (check_var('SLE_PRODUCT', 'hpc'))) {
```

Comment:
We can only  use check_var('CROSS_PRODUCT_MIGRATION', 'from_hpc') || check_var('SLE_PRODUCT', 'hpc') condition.

Reply:
Yes, I updated the pr and re-run the verification run, it works fine, thank you.